### PR TITLE
Adam

### PR DESCRIPTION
**Pull request by fadehelix**

### CODE REVIEW

File: lista-roberta.md [hunk 1]
```diff
@@ -32,3 +32,6 @@
 # Inne
 - [Donella Meadows - Myślenie systemowe](https://www.goodreads.com/book/show/3828902-thinking-in-systems)
 - [Five Agile Metrics you won't hate](https://www.atlassian.com/agile/project-management/metrics)
+
```
Comment (alewecka) [inline lista-roberta.md:35]: Lalalala

File: lista-roberta.md [hunk 2]
```diff
@@ -32,3 +32,6 @@
...
+
```
Comment (JAdrianek) [inline lista-roberta.md:36]: ## Adrian komentarz

Comment (alewecka) [inline lista-roberta.md:36]: Test

**Review by alewecka — APPROVED** [review]

**Comment by JAdrianek** [comment]
great job!

**Pull request merged**